compiler/kphp2cpp.cpp

```diff
@@ -261,6 +261,8 @@ int main(int argc, char *argv[]) {
              "php-code-version", "KPHP_PHP_CODE_VERSION", "unknown");
   parser.add("C++ compiler for building the output binary", settings->cxx,
              "cxx", "KPHP_CXX", get_default_cxx());
+  parser.add("Directory for compiler toolchain. Will be passed to compiler through -B option", settings->compiler_toolchain_dir,
```

Comment:
"for C++ compiler toolchain"? Not to be confused with kphp compiler itself.
BTW, maybe to name this option `cxx-toolchain-dir`?

Reply:
> "for C++ compiler toolchain"? Not to be confused with kphp compiler itself.
Done.
> BTW, maybe to name this option cxx-toolchain-dir?
I'm doubt in. cxx is not compiler, I guess it bad naming.

Reply:
But from the other point of view, `cxx` means "C++ compiler" (g++, or clang, or /usr/bin/c++, etc). We use `KPHP_CXX`, `KPHP_EXTRA_CXXFLAGS` and others. CMake also uses such naming (example: `-DCMAKE_CXX_COMPILER`, `-DCMAKE_CXX_COMPILER_LAUNCHER`). I think, "cxx" would be exactly what in means here

Reply:
Ok, done.